src/main/java/prestudy/team4/board/domain/PostLike.java

```diff
@@ -4,17 +4,26 @@
 import jakarta.persistence.*;
 import lombok.*;
 
-import java.time.LocalDateTime;
-
 @Getter
 @NoArgsConstructor @AllArgsConstructor @Builder
 @Entity
-@Table(name = "post_likes")
+@Table(
+        name = "likes",
+        uniqueConstraints = @UniqueConstraint(
+                name = "uk_likes_post_user",
+                columnNames = {"postId", "userId"}
+        )
+)
 public class PostLike {
 
-    @EmbeddedId
-    private PostLikeId id;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long likeId;
+
+    @Column(nullable = false)
+    private Long postId;
+
+    @Column(nullable = false)
+    private Long userId;
```

Comment:
복합키 -> 단일키로 바꿔도 FK 관계는 존재합니다! `postId`와 `userId`는 그냥 숫자로 저장되고 있는데, 게시글이 삭제되었을 때  좋아요는 남아있는... 이상한 상황이 남게될 수 있어요.

```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "post_id", nullable = false)
    private Post post;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id", nullable = false)
    private User user;
```